src/app/dashboard/revalidation-form/revalidation-form.component.scss

```diff
@@ -0,0 +1,9 @@
+.revalidation-form {
+  min-width: 150px;
+  max-width: 500px;
+  width: 100%;
```

Comment:
this width could be removed and the `w-100` class can be leveraged instead

Reply:
done.